### PR TITLE
Header

### PR DESCRIPTION
**Pull request by kelvin8773**
* add SVG logo
* arrange layout of  navbar items;
* add react-router to header navbar;
* Home / Search links work;

### CODE REVIEW

File: src/navbar/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </Link>
+      </li>
+
+      <li>
+        <Link to={{
+          pathname: '/',
+          hash: '#how-it-work',
+        }}
+        >
+          How it works
```
Comment (jkettmann) [inline src/navbar/index.js]: The color of the links is not matching

File: src/navbar/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Link,
+} from 'react-router-dom';
+import styles from './Navbar.module.scss';
+
+const Navbar = () => (
+  <div className={styles.Navbar}>
+    <Link to="/">
+      <img
```
Comment (jkettmann) [inline src/navbar/index.js]: I guess it's easier to have the logo.svg inside the src/navbar directory. Then you can simply

```
import Logo from './logo.svg'
```

and render it like `<Logo className="..." />

File: src/stylesheets/_fonts.scss [hunk 3]
```diff
@@ -1,18 +1,26 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:100,200i,300,400,500,600,700&display=swap');
```
Comment (jkettmann) [inline src/stylesheets/_fonts.scss]: I don't think we use all of the font-weights

File: src/main/index.js [hunk 4]
```diff
@@ -7,9 +7,10 @@ import {
 
 import Home from './home';
 import Search from './search';
+import styles from './Main.module.scss';
 
 const Main = () => (
-  <div className="Main text-center">
+  <div className={styles.Main}>
```
Comment (jkettmann) [inline src/main/index.js]: I would use consistent naming for your class names. Here you have `Main`, somewhere else `about` or `how_it_works`. In general I would suggest using camel casing, meaning `main` and `howItWorks`

File: src/stylesheets/_fonts.scss (hunk 3 above)
Comment (kelvin8773) [reply src/stylesheets/_fonts.scss]: remove some of them, just keep 400, 500, 700

File: src/navbar/index.js (hunk 2 above)
Comment (kelvin8773) [reply src/navbar/index.js]: Ok, good, I don't how this before, it is a good practice.

File: src/main/index.js (hunk 4 above)
Comment (kelvin8773) [reply src/main/index.js]: alright, updated, now should be consistent

File: src/navbar/index.js (hunk 1 above)
Comment (kelvin8773) [reply src/navbar/index.js]: updated it.

File: src/navbar/Navbar.module.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
+.Navbar {
+    height: 100px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 80px;
+
+    ul {
```
[the remark this answers is not in the document]
Comment (kelvin8773) [reply src/navbar/Navbar.module.scss]: yes, it is good practice, updated it now.

File: src/navbar/Navbar.module.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.Navbar {
```
[the remark this answers is not in the document]
Comment (kelvin8773) [reply src/navbar/Navbar.module.scss]: update it now.

**Comment by kelvin8773** [comment]
Thanks for the review, I already update the repo accordingly, please check again, thanks!

File: src/navbar/index.js (hunk 2 above)
Comment (jkettmann) [reply src/navbar/index.js]: Seems like my comment was cut off. Importing is correct now. But you can simply render the logo like this

```
import { ReactComponent as Logo } from './logo.svg';

<Logo className={styles.linkLogo} />
```

File: src/navbar/navbar.module.scss [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    align-items: center;
+    padding: 0 80px;
+}
+
+.linkItems{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    .linkItem{
```
Comment (jkettmann) [inline src/navbar/navbar.module.scss]: Move `.linkItem` to one level up, meaning having all of the classes at the same level like below. Then you don't have to use a string as className like here `<Link to="/" className="linkLogo">`

```
.navbar {...}

.linkItems {...}

.linkItem {...}
```

File: src/app.module.scss [hunk 8]
```diff
@@ -0,0 +1,4 @@
+.app {
+    width: 1440px;
```
Comment (jkettmann) [inline src/app.module.scss]: not sure, but you might need to use `margin: auto` here to center the app content

Comment (kelvin8773) [reply src/app.module.scss]: yes, right.

File: src/navbar/navbar.module.scss (hunk 7 above)
Comment (kelvin8773) [reply src/navbar/navbar.module.scss]: updated the logo render, great trick, I never learn that if you didn't tell me.

File: src/navbar/navbar.module.scss [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+.linkItems{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+}
+
+.linkItem{
+    margin-left: 25px;
```
Comment (jkettmann) [inline src/navbar/navbar.module.scss]: Unfortunately the layout doesn't match the design yet. You can see a slight blur at the links, especially the `Search` link. When you increase `margin-left` to `26px` that's solved ;-)

image

Comment (kelvin8773) [reply src/navbar/navbar.module.scss]: image
but how come in ziplin it show 25px?

Comment (kelvin8773) [reply src/navbar/navbar.module.scss]: ok, it does look closer to ziplin design with 26px.

**Pull request merged**